src/core/ext/transport/cronet/transport/cronet_transport.cc

```diff
@@ -1090,22 +1093,30 @@ static enum e_op_result execute_stream_op(struct op_and_state* oas) {
       grpc_slice_buffer write_slice_buffer;
       grpc_slice slice;
       grpc_slice_buffer_init(&write_slice_buffer);
-      if (1 != stream_op->payload->send_message.send_message->Next(
-                   stream_op->payload->send_message.send_message->length(),
-                   nullptr)) {
-        /* Should never reach here */
-        GPR_ASSERT(false);
-      }
-      if (GRPC_ERROR_NONE !=
-          stream_op->payload->send_message.send_message->Pull(&slice)) {
-        /* Should never reach here */
-        GPR_ASSERT(false);
+      while (write_slice_buffer.length <
+             stream_op->payload->send_message.send_message->length()) {
+        if (1 != stream_op->payload->send_message.send_message->Next(
+                     stream_op->payload->send_message.send_message->length(),
+                     nullptr)) {
+          /* Should never reach here */
+          GPR_ASSERT(false);
+        }
+        /*
+          TODO(roth): When we add support for incremental sending,this code will
```

Comment:
Please move this comment up above the call to `Next()`.

Also, please use C++-style comment syntax.

Reply:
done